packages/fxa-settings/src/components/PasswordStrengthInline/index.tsx

```diff
@@ -87,6 +90,19 @@ export const PasswordStrengthInline = ({
             </FtlMsg>
           </li>
         )}
+        {isUnconfirmed !== undefined && passwordFormType === 'signup' && (
```

Comment:
This criteria shouldn't be shown for signup/non-sync flow that only has one password field.

Reply:
Pushed a commit.  Thank you for catching that.